internal/service/projectipaccesslist/resource_schema.go

```diff
@@ -56,7 +56,6 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 			},
 			"aws_security_group": schema.StringAttribute{
 				Optional: true,
-				Computed: true,
```

Comment:
I am hesitant on the removal of `Computed: true` for both `aws_security_group` and `comment` due to risk of breaking changes. I would strictly preserve the current (incorrect) behaviour of `getting a null value but setting it as an empty string` to be on the safer side. 

We can create a follow up ticket to address this removal and flag it as a candidate for next major release (v3.0). We have a jira tag to keep track of these candidates (`terraform-provider-3.x`).